### DISCUSSION TITLE
Allow Lower TDP for custom power profile

### DISCUSSION THREAD

**Discussion by Layen03** [BartoszCichecki/LenovoLegionToolkit#1431] [Ideas]
Hello, first i would like to say that i'm so glad that i found this software, amazing work.

For me the only thing missing is the ability to set a max TDP lower than what is possible now for the custom power profile mode.
Indeed, i made some tests with UX86TU and my processor performs roughly the same at 30 and 50W in game (R7 8845HS) minus the heat. 
I would like to know if it was possible to implement this feature as this would allow to have an extremely fine tuned custom mode, high perf low noise and low temp (30W CPU and max W GPU) or extremely low power high battery life custom mode (hybrid mode 10W CPU) for low intensity tasks.

 Do you think this would be possible ? I would be happy to test it out.

**Comment by BartoszCichecki** [comment]
You can already now override min/max value for custom mode sliders using offsets (they are at the bottom of custom mode settings). It isn't guaranteed that EC will respect values beyond the default range. So it might be possible, but is also may not work at all.

**Comment by Layen03** [comment]
Oh i didn't understand what these were for, my bad, gonna try that thank you very much

Edit : Works perfectmy fine for me, really really happy ! Thank you !
